django_filters/filterset.py

```diff
@@ -357,6 +374,16 @@ def get_filters(cls):
         filters.update(cls.declared_filters)
         return filters
 
+    @classmethod
+    def handle_unrecognized_field(cls, field_name, message):
+        behavior = cls._meta.unknown_field_behavior
+        if behavior == UnknownFieldBehavior.RAISE:
+            raise AssertionError(message)
+        elif behavior == UnknownFieldBehavior.WARN:
+            warnings.warn(f"Unrecognized field type for '{field_name}'. Field will be ignored.")
+        elif behavior == UnknownFieldBehavior.IGNORE:
+            pass
```

Comment:
I think we **still** need the unknown value branch here (as well) since the test using the string should error.

Reply:
Fair enough, I added it again + test